environments/dev-env/README.md

```diff
@@ -0,0 +1,515 @@
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/wls_ubuntu_platform.md)
+
+##  Contributing
+
+See the [contributing instructions](CONTRIBUTING.md) for detailed instructions on how to deploy and get started with our project.
+
+## Example Scenario
+
+See the [example scenario instructions](docs/example_stages.md) to quickly deploy a kcp-edge demo environment with two sample workloads and two edge clusters. 
+
+## Quickstart
+
+### 1. Download the kcp-edge binaries:
+
+
+```shell
+VERSION=0.1.0 # choose the latest version (without v prefix)
+OS=darwin   # linux or darwin
+ARCH=amd64  # arm64 or amd64
+curl -sSfL "https://github.com/kcp-dev/edge-mc/releases/download/v${VERSION}/kcp-edge_v${VERSION}_${OS}_${ARCH}.tar.gz" > kcp-edge.tar.gz
+```
+
+Add the kcp edge `/bin` directory to your $PATH:
+
+```shell
+tar -xvf kcp-edge.tar.gz
+export PATH="$PATH:$(pwd)/bin"
+```
+
+
+### 2. Create your own edge infrastructure (edge clusters) - Bring Your Own Cluster (BYOC)
+
+Create your edge cluster or bring your own k8s edge cluster. In this example, we use [kind](https://kind.sigs.k8s.io/) to create an edge cluster that we name “florin”:
+
+```shell
+kind create cluster --name florin
+```  
+
+### 3. Deploy the kcp-edge platform:
+
+  a) Download kcp binaries for your platform:
+
+  ```shell
+  mkdir kcp
+  cd kcp
+  VERSION=0.11.0 # choose the latest version (without v prefix)
+  OS=darwin   # linux or darwin
+  ARCH=amd64  # arm64 or amd64
+  curl -sSfL "https://github.com/kcp-dev/kcp/releases/download/v${VERSION}/kcp_${VERSION}_${OS}_${ARCH}.tar.gz" > kcp.tar.gz
+  curl -sSfL "https://github.com/kcp-dev/kcp/releases/download/v${VERSION}/kubectl-kcp-plugin_${VERSION}_${OS}_${ARCH}.tar.gz" > kubectl-kcp-plugin.tar.gz
+  ```
+
+  Extract kcp and kubectl-kcp-plugin and place all the files in the bin directories somewhere in your $PATH. For example:
+
+  ```shell
+  tar -xvf kcp.tar.gz
+  tar -xvf kubectl-kcp-plugin.tar.gz
+  export PATH="$PATH:$(pwd)/bin"
+  ```
+
+  b) Start kcp:
+
+  ```shell
+  kcp start >& kcp_log.txt &
+  ```
+  OR
+    
+  If you need to connect to a remote k8s edge cluster, you can bind kcp to the public interface of your virtual machine:
+
+  ```shell
+  kcp start --bind-address <ip-address> >& kcp_log.txt &
+  ```
+    
+  This uses default 6443 port, just bind to a specific address
+    
+  Set your KUBECONFIG variable to the file in `.kcp/admin.kubeconfig` generated by kcp during startup:
+
+  ```shell
+  export KUBECONFIG="$(pwd)/.kcp/admin.kubeconfig"
+  ```
+
+  c) Deploy kcp-edge infra:
+         
+  Run the following command to start kcp-edge:
+
+  ```bash
+  bash <(curl -s https://raw.githubusercontent.com/dumb0002/edge-mc/user-dev-kit/environments/dev-env/kcp-edge.sh) start --user kit
+  ```
```

Comment:
where is this file (`kcp-edge.sh`) ? it's not part of the PR or in the main repo.

Reply:
alas, it has not merged yet.

Reply:
The script is in this PR: https://github.com/kcp-dev/edge-mc/pull/347

Reply:
Since we are focusing on the user experience before the contributor experience, a script needed by users should not be held up in a PR for the contributor experience.

Reply:
I think the reference to #347 was wrong, isn't the script in #354 ?